readme.md

```diff
@@ -169,6 +169,8 @@ It exports the additional types [`Options`][api-options] and
 To type `file.data.matter`, you can augment `DataMap` from `vfile` as follows:
 
 ```ts
+import {DataMap} from 'vfile'
+
```

Comment:
I think the problem is that the file needs to be a module, not a script. Adding an import or export fixes that. The most elegant fix IMO is:

```suggestion
export {}

```